Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -290,7 +290,8 @@ def main() -> None:
     credentials = params.get('credentials')
     proxy = params.get('proxy', False)
     insecure = params.get('insecure', False)
-    first_fetch_datetime = arg_to_datetime(params.get('first_fetch'))
+    first_fetch_param = params.get('first_fetch')
+    first_fetch_datetime = arg_to_datetime(first_fetch_param) if first_fetch_param else None
```

Comment:
```suggestion
    first_fetch_datetime = arg_to_datetime(first_fetch_param, required=False)
```

Reply:
It will not work. Default value for required is False. `arg_to_datetime` checks if arg is None and not required. In the bug the issue is that `first_fetch_param` is "" and not None.